README.md

```diff
@@ -140,37 +171,45 @@ services to be up and running.
 
 Purge all data:
 
-```
-$ docker-compose exec sparql ./vendor/bin/robo purge
+```bash
+docker-compose exec sparql ./vendor/bin/robo purge
 ```
 
 Or, if you can run commands on your host machine:
 
-```
-$ ./vendor/bin/run sparql:purge
+```bash
+./vendor/bin/run sparql:purge
 ```
 
 Import default data:
 
-```
-$ docker-compose exec sparql ./vendor/bin/robo import
+```bash
+docker-compose exec sparql ./vendor/bin/robo import
 ```
 
 Or, if you can run commands on your host machine:
 
-```
-$ ./vendor/bin/run sparql:import
+```bash
+./vendor/bin/run sparql:import
 ```
 
 Reset all data, i.e. run purge and import:
 
-```
-$ docker-compose exec sparql ./vendor/bin/robo purge
-$ docker-compose exec sparql ./vendor/bin/robo import
+```bash
+docker-compose exec sparql ./vendor/bin/robo purge
+docker-compose exec sparql ./vendor/bin/robo import
 ```
 
 Or, if you can run commands on your host machine:
 
+```bash
+./vendor/bin/run sparql:reset
 ```
-$ ./vendor/bin/run sparql:reset
-```
+
+## Contributing
+
+Please read [the full documentation](https://github.com/openeuropa/openeuropa) for details on our code of conduct, and the process for submitting pull requests to us.
+
+## Versioning
+
+We use [SemVer](http://semver.org/) for versioning. For the available versions, see the [tags on this repository](https://github.com/openeuropa/drupal_module_template/tags).
```

Comment:
Wrong link, it must refer to oe_content .

Reply:
Fixed.